### PR TITLE
add js extensions

### PR DESCRIPTION
**Pull request by Rich-Harris**
Via #236 – this PR adds .js extensions to all local module imports

### CODE REVIEW

**Comment by Victorystick** [comment]
LGTM! :+1:

**Comment by TrySound** [comment]
@Rich-Harris `es6-promise/lib/es6-promise/promise` not local but still file, not folder or package name

**Comment by TrySound** [comment]
Oops, sorry :)

**Comment by TrySound** [comment]
But `import { parse } from 'acorn/src/index';`

**Comment by Victorystick** [comment]
@TrySound Nice catch!

File: src/Module.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import { parse } from 'acorn/src/index';
```
Comment (TrySound) [inline src/Module.js]: here

Comment (TrySound) [inline src/Module.js]: Here

File: src/finalisers/amd.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { getName, quoteId } from '../utils/map-helpers';
```
Comment (TrySound) [inline src/finalisers/amd.js]: Here

File: src/finalisers/iife.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { blank } from '../utils/object';
 import { getName } from '../utils/map-helpers';
```
Comment (TrySound) [inline src/finalisers/iife.js]: Here

File: src/finalisers/umd.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { blank } from '../utils/object';
 import { getName, quoteId, req } from '../utils/map-helpers';
```
Comment (TrySound) [inline src/finalisers/umd.js]: Here

**Comment by Rich-Harris** [comment]
Thanks @TrySound – that's what I get for doing a quick find-and-replace! Updated

**Pull request merged**